lib/coinbase/exchange/api_object.rb

```diff
@@ -15,7 +15,7 @@ def update(data)
       def format(var)
         return if var.nil?
         # Looks like a number or currency
-        if var =~ /^.{0,1}\s*[0-9,]*\.{0,1}[0-9]*$/
+        if var.respond_to?(:=~) && var =~ /^.{0,1}\s*[0-9,]*\.{0,1}[0-9]*$/
```

Comment:
```suggestion
        if var&.match?(/^.{0,1}\s*[0-9,]*\.{0,1}[0-9]*$/)
```

Reply:
I'll keep the respond_to? because it looks like this method accepts any kind of var argument. I think safer with this check.